app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1317,23 +1315,28 @@ class PatientRepositoryAndroidTest {
         creationFacilityUuid = facilityUuid,
         status = Scheduled
     )
-    val recentPatient2 = savePatientWithAppointment(
-        facilityUuid = facilityUuid,
-        creationFacilityUuid = facilityUuid,
-        status = Cancelled
-    )
-    val recentPatient3 = savePatientWithAppointment(
-        facilityUuid = facilityUuid,
-        creationFacilityUuid = facilityUuid,
-        status = Visited
-    )
 
     val recentPatients = patientRepository
         .recentPatients(facilityUuid)
         .blockingFirst()
     assertThat(recentPatients).isEqualTo(listOf(recentPatient1))
   }
 
+  @Test
+  fun verify_only_list_of_colonies_or_villages_are_received_from_patient_address() {
+    val address1 = testData.patientAddress(colonyOrVillage = "Sharma Courts")
+    val address2 = testData.patientAddress(colonyOrVillage = "7111 Jain Mountains")
+    val address3 = testData.patientAddress(colonyOrVillage = "138, residency road")
+
+    val addresses = listOf(address1, address2, address3)
+    database.addressDao().save(addresses)
+
+    val expectedListOfColoniesOrVillages = listOf(address1.colonyOrVillage, address2.colonyOrVillage, address3.colonyOrVillage)
+
+    val listOfColoniesOrVillages = patientRepository.allColoniesOrVillagesInPatientAddress()
+    assertThat(listOfColoniesOrVillages).isEqualTo(expectedListOfColoniesOrVillages)
+  }
```

Comment:
Hmm, are we populating this from the addresses that already exist? I was under the impression that we will be fetching a list from the server in order to encourage a set of known names. Was this changed?

Reply:
We're using the current sync region’s patient addresses to populate the village names. Here's the updated conversation : https://simpledotorg.slack.com/archives/C01FN9GCNAJ/p1611753504014400?thread_ts=1611542193.001000&cid=C01FN9GCNAJ